public/modules/custom/grants_handler/src/Processor/NumberProcessor.php

```diff
@@ -13,10 +13,11 @@ class NumberProcessor {
    * Process number fields to allow . or , and convert them for validators.
    */
   public static function process(&$element, FormStateInterface $form_state, &$complete_form) {
-    $value = trim($element['#value']);
+    $valueFromElement = $element['#value'] ?? NULL;
```

Comment:
Seuraavalla rivillä oleva trim muuttaa NULLin tyhjäksi merkkijonoksi. Mutta jos muuttuja olisi koko ajan stringi niin koodia lukiessa koodaajaan ei tarvitse ruveta pohtimaan trimmin toimintaa.

```suggestion
    $valueFromElement = $element['#value'] ?? '';
```